distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+
+The following table shows the format to use for each property type supported.
+
+[cols="1,2,4" options="header"]
+|===
+
+|Type
+|Format
+|Example
+
+|Service PID
+|`service.pid = "servicePid"`
+|`service.pid = "org.codice.ddf.security.policy.context.impl.PolicyManager"`
+
+|Factory PID
+|`service.factoryPid = "serviceFactoryPid"`
+|`service.factoryPid = "Csw_Federated_Source"`
+
+|Strings
+|`name = "value"`
+|`name = "john"`
+
+|Booleans
+|`name = B"true\|false"`
+|`authorized = B"true"`
+
+|Integers
+|`name = I"value"`
+|`timeout=I"60"`
+
+|Longs
+|`name = L"value"`
+|`diameter = L"10000"`
+
+|Floats
+|`name = F"value"`
+|`cost = F"10.50"`
+
+|Doubles
+|`name = D"value"`
+|`latitude = D"45.0234"`
+
+|Lists of Strings
+|`name = [ "value1", "value2", ... ]`
+|`complexStringArray = [ "{\"url\"\ \"http://test.sample.com\"\ \"layers\"\ [\"0\"]\ {\"VERSION\"\ \"1.1\|1.2\"\ \"image/png\"}\ \"beta\"\ 1}", "{\"url\"\ \"http://test.sample.com"\ 0.5}", "/solr\=SAML\|PKI\|basic", "/security-config\=SAML\|basic" ]`
+
+|Lists of Integers
+|`name = I[ "value1", "value1", ... ]`
+|`sizes = I[ "10", "20", "30" ]`
+
+|===
+
+[NOTE]
+====
+* Lists of values can be prefixed with any of the supported types (`B`, `I`, `L`, `F` or `D`)
+* To prevent any configuration issues, the `=` signs used in values should be escaped using a backslash (`\`)
+* Boolean values will default to `false` if any value other than `true` is provided
+* Escape character in values must be used for double quotes (`"`) and spaces, but cannot be used with `{` `}` or `[` `]` pairings
+====
+
+.Sample configuration file
+[source,java,linenums]
+----
+service.pid="org.codice.ddf.security.policy.context.impl.PolicyManager"
+
+authenticationTypes=["/\=SAML|GUEST","/admin\=SAML|basic","/system\=basic","/solr\=SAML|PKI|basic","/sources\=SAML|basic","/security-config\=SAML|basic","/search\=basic"]
+
+realms=["/\=karaf"]
+
+requiredAttributes=["/\=","/admin\={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role\=admin}","/solr\={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role\=admin}","/system\={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role\=admin}","/security-config\={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role\=admin}"]
+
+whiteListContexts=["/services/SecurityTokenService","/services/internal/metrics","/services/saml","/proxy","/services/csw"]
+----
+
+===== Editing HTTP Ports for Multiple Local ${branding} Nodes
+
+Edit the port numbers in the files in the ${branding} install folder.
+
+[cols="4" options="header"]
+|===
+
+|File to Edit
+|Property(ies)
+|Original Value
+|Example of New Value
+
+|`bin/karaf.bat`
+|`address`
+|`5005`
+|`5006`
+
+.2+|`etc/org.apache.karaf.management.cfg`
+|`rmiRegistryPort`
+|`1099`
+|`1199`
+
+
+|`rmiServerPort`
+|`44444`
+|`44445`
+
+.2+|`etc/system.properties`
+|`httpsPort`,`port`
+|`8993`
+|`8994`
+
+|`httpPort`
+|`8181`
+|`8281`
+
+|===
+
+==== Editing ${branding} Web Service Providers Configuration Files
```

Comment:
Not sure why we have this section here. If we don't have any useful information to put here we should delete it.